### DISCUSSION TITLE
Force debounced input field to make update request?

### DISCUSSION THREAD

**Discussion by henkpoldergraaf** [livewire/livewire#4997] [1. Help]
Hi there,

I was wondering if there is a way force debounced input fields to make the request early. I am working with a rich text editor which I do not want to fire on every keystroke, so I added the debounce modifier to it (1s). The issue I am experiencing is that when you change/update/leave the page the rich text editor is on before the timer ends, the input is not sent to the backend. 

Also if I reinitialise a Livewire component with new data before the debounced timer has end, the debounced (old) data from the previous editor gets send to the newly initialised component.

So my idea was to trigger the sending if the request myself when needed, but I could not find anything similar to it. Is this even possible you think?

**Comment by joshhanley** [comment]
@henkpoldergraaf you could create a custom input using Alpine and handle all of it there, then you can interact with Livewire's javascript object to call updates whenever you require. See here for more info https://laravel-livewire.com/docs/2.x/alpine-js#interacting-with-livewire-from-alpine

Hope this helps!
